src/AzureExtension/DevBox/DevBoxProvider.cs

```diff
@@ -191,6 +195,24 @@ public IAsyncOperation<ComputeSystemsResult> GetComputeSystemsAsync(IDeveloperId
         }).AsAsyncOperation();
     }
 
+    // Handle common WAM errors
+    // Reference: https://learn.microsoft.com/en-us/entra/msal/dotnet/advanced/exceptions/wam-errors
+    private string SimplifyWAMError(string errorMessage)
+    {
+        if (errorMessage.Contains("3399614467"))

```

Comment:
Are those error codes anywhere in the output/logs/telemetry? If not they should be added. Having only text messages without error codes makes investigations more difficult:

- Messages can be in different languages: it’s more difficult to search and aggregate.
- Filtering/searching by error code is simpler and more accurate.
- Error codes can be used in telemetry and RQVs to monitor specific issues.

Reply:
That's a good point. Changed.